cypress/support/testData.js

```diff
@@ -0,0 +1,9 @@
+export const user = {
+    username: 'KillerPencil',
+    password: '123456Test!'
+}
+
+export const book = {
+    title: 'Speaking JavaScript',
+    description: 'Like it or not'
+}
\ No newline at end of file
```

Comment:
It's a good practice to have a newline at the end of file to avoid unexpected behavior when concatenating or processing files. Consider adding a newline at the end of the file. 😊